lib/THC/CMakeLists.txt

```diff
@@ -198,10 +199,18 @@ INSTALL(FILES
           generic/THCTensorCopy.c
           generic/THCTensorCopy.cu
           generic/THCTensorCopy.h
+          generic/THCTensorMasked.h
+          generic/THCTensorMasked.cu
           generic/THCTensorMath.h
           generic/THCTensorMath.cu
+          generic/THCTensorMathCompare.h
+          generic/THCTensorMathCompare.cu
+          generic/THCTensorMathCompareT.h
+          generic/THCTensorMathCompareT.cu
           generic/THCTensorMathPairwise.h
           generic/THCTensorMathPairwise.cu
           generic/THCTensorMathPointwise.h
           generic/THCTensorMathPointwise.cu
+          generic/THCTensorMathReduce.h
+          generic/THCTensorMathReduce.cu
           DESTINATION "${THC_INSTALL_INCLUDE_SUBDIR}/THC/generic")
```

Comment:
MathBlas files are missing from the export, making CUNN fail.
Here : https://github.com/borisfom/cutorch/commit/ccff07f0616e1b5b5767eeb8b01e659f530c0ac9